sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -515,11 +515,24 @@ def request_id(self):
 class CertificatePolicy(object):
     """Management policy for a certificate.
 
-    :param attributes: the certificate attributes.
+    :param attributes: The certificate attributes. These are set by the service.
```

Comment:
The docs here are for users doing `CertificatePolicy(...)`, so we shouldn't document (or include in the signature) arguments they shouldn't pass, like `attributes`.

Reply:
sounds good, I also got rid of the certificate_id field because that's service set as well.

Reply:
thanks, I have a separate PR out for getting rid of all arguments users shouldn't pass. thansk!